ecommerce-ui/src/Components/Cart/Cart.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { useEffect } from "react";
+import PropTypes from "prop-types";
+import CartCard from "./CartCard";
+import CartSummary from "./CartSummary";
+import "./Cart.css";
+
+export default function Cart({
+  cartItems,
+  onRemoveFromCartHandler,
+  locationData,
+}) {
+  useEffect(() => {
+    window.scrollTo(0, 0);
+  }, []);
```

Comment:
I feel like this should be a reusable hook, considering how often it needs to be used with react-router 😅